src/parse/README.md

```diff
@@ -0,0 +1,18 @@
+# Package parse
+
+*This readme is a stub. Please raise an issue on this repo if you would like something expanded on.*
```

Comment:
I dont think this clarification is required. We should expand the docs whenever possible though.